lib/Services/RetrieveMetadata.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * ownCloud - mediametadata
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the COPYING file.
+ *
+ * @author Jalpreet Singh Nanda (:imjalpreet) <jalpreetnanda@gmail.com>
+ * @copyright Jalpreet Singh Nanda (:imjalpreet) 2016
+ */
+
+namespace OCA\MediaMetadata\Services;
+
+
+use OCP\IDBConnection;
+
+class RetrieveMetadata {
+
+	protected $dbConnection;
+
+	/**
+	 * @param IDBConnection $connection
+	 */
+	public function __contruct(IDBConnection $connection){
+		$this->dbConnection = $connection;
+	}
+
+	public function retrieve($fileList) {
```

Comment:
We need some PHPdoc so that we understand what fileList is. It is a list of IDs.